### DISCUSSION TITLE
Compatibility of flash-admin url = / "

### DISCUSSION THREAD

**Discussion by jackadam1981** [tiangolo/uwsgi-nginx-flask-docker#369] [Questions] [answered]
The relevant codes are as follows
https://github.com/jackadam1981/test_flask_admin.git

If flash-adimn is used, the user-defined URL address is url = '/',
It works well under windows and Linux.
The static file of flash-admin cannot be accessed normally in the container.

    #admin = Admin(app,url='/')    
    admin = Admin(app)    
Correct record
 10.0.0.210 - - [14/Mar/2021:01:47:28 +0000] "GET /admin/static/bootstrap/bootstrap2/swatch/default/bootstrap.min.css?v=2.3.2 HTTP/1.1" 304 0 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.82 Safari/537.36" "-"
[pid: 16|app: 0|req: 3/5] 10.0.0.210 () {48 vars in 1001 bytes} [Sun Mar 14 01:47:28 2021] GET /admin/static/bootstrap/bootstrap2/swatch/default/bootstrap.min.css?v=2.3.2 => generated 0 bytes in 1 msecs (HTTP/1.1 304) 4 headers in 183 bytes (0 switches on core 0)

    admin = Admin(app,url='/')    
    #admin = Admin(app)    
Record of errors
2021/03/14 01:32:12 [error] 14#14: *3 open() "/app/static/admin/bootstrap/bootstrap2/js/bootstrap.min.js" failed (2: No such file or directory), client: 10.0.0.210, server: , request: "GET /static/admin/bootstrap/bootstrap2/js/bootstrap.min.js?v=2.3.2 HTTP/1.1", host: "10.0.0.10", referrer: "http://10.0.0.10/"
2021/03/14 01:32:12 [error] 14#14: *1 open() "/app/static/admin/vendor/moment.min.js" failed (2: No such file or directory), client: 10.0.0.210, server: , request: "GET /static/admin/vendor/moment.min.js?v=2.22.2 HTTP/1.1", host: "10.0.0.10", referrer: "http://10.0.0.10/"

**Comment by tiangolo** [comment, marked as answer]
Hey, please create a small `Dockerfile`, the minimal dependencies, and a minimal app file, that way I can try to replicate your error.

Otherwise I wouldn't be able to help as I don't have enough information.

**Comment by jackadam1981** [comment]
it's along time ago.
the file is delete.
i'll try do aging tomorr.
